py3status/modules/scratchpad_async.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display the amount of windows and indicate urgency hints on scratchpad (async).
+Display number of windows in scratchpad asynchronously and indicate urgency hints.
```

Comment:
Display number of windows in scratchpad and indicate urgency hints. (async) 
And put (async) on window_title_async.py too to balance it out?